### DISCUSSION TITLE
夸克网盘上传up status: 409

### DISCUSSION THREAD

**Discussion by HaishuMa** [AlistGo/alist#4309] [Q&A]
up status: 409, error: <?xml version="1.0" encoding="UTF-8"?> <Error> <Code>PartAlreadyExist</Code> <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message> <RequestId>6453BEE1DD7F7835367B41BC</RequestId> <HostId>ccp-sz3-sz-1564369074.oss-cn-shenzhen.aliyuncs.com</HostId> <PartEtag>"04CC17F50D92360F083913EAC012A362"</PartEtag> <PartNumber>42</PartNumber> <EC>0042-00000108</EC> </Error>

**Comment by gofunbox** [comment]
+1

**Reply by gofunbox** [reply]
单个单个传好像会降低频率~~~

**Comment by 929126387** [comment]
+1

**Comment by ashesofdream** [comment]
有解决方法了吗

**Comment by joewongex** [comment]
+1
